bofire/benchmarks/LookUpTableBenchmark.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pandas as pd
+
+from bofire.benchmarks.benchmark import Benchmark
+from bofire.data_models.domain.api import Domain
+
+
+class LookUpTableBenchmark(Benchmark):
```

Comment:
```suggestion
class LookupTableBenchmark(Benchmark):
```

It is one word "lookup".